.env.development

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_URL=https://forma-devnet-modularspring.onrender.com/
```

Comment:
Why did you call this `NEXT_PUBLIC_API_URL` ? That is a very specific env var for nextjs apps and not what this is. Please name appropriately.